autometa/common/exceptions.py

```diff
@@ -87,8 +87,12 @@ class SamtoolsSortError(AutometaException):
         Base class for other exceptions
     """
 
-    def __init__(self, value):
-        self.value = value
+    def __init__(self, cmd, err):
+        self.cmd = cmd
+        self.err = err
+
+    def __str__(self):
+        return f"Error while running external utility \n cmd: {self.cmd} \n error: {self.err}"
```

Comment:
Minor nitpick. But could you swap the error and cmd display? I think more often we will want to look at the command run first before we dive into the traceback information